### ISSUE TITLE
Declaration of ccxt\kuna::fetch_open_orders($symbol, $params = Array) should be compatible with ccxt\Exchange::fetch_open_orders($symbol = NULL, $params = Array)

### ISSUE THREAD

**Issue by rubobaquero** [ccxt-dev/ccxt#359]
I am receiving this warning (version 1.9.212):

```
PHP Warning:  Declaration of ccxt\kuna::fetch_open_orders($symbol, $params = Array) should be compatible with ccxt\Exchange::fetch_open_orders($symbol = NULL, $params = Array) in /XXXXX/ccxt.php on line 17420
```
It seems that the only difference between both declarations is that " = NULL" is missing on the kuna class.

**Comment by kroitor** [comment]
Thx! Missed it somehow. The fix is on the way!

**Comment by kroitor** [comment]
The fix is there in version 1.9.214. Thanks again!
